app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RelationshipsController < ApplicationController
+  before_filter :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.delete if relationship.follower == current_user
```

Comment:
Sorry, I typed `delete` the second time too, but it should have been `destroy`. The link was correct though. I have fixed the original comment.